### ISSUE TITLE
Adaptation to classification and detection problem

### ISSUE THREAD

**Issue by aungriah** [SimonVandenhende/Multi-Task-Learning-PyTorch#9]
Hi @SimonVandenhende,

Thank you for sharing your code! I was wondering if you had tested the network also for other tasks (I am trying to train for joint 3D-object detection and classification & road lane detection and would like to know if you thought this is suitable with MTI-NET)

Thank you!

**Comment by SimonVandenhende** [comment]
Hi

Thanks for the interest. MTI-Net is mostly focussed on tackling multiple dense prediction tasks.
I have no experience in using it for other tasks, like the ones you are suggesting.
Note for example that image classification tasks are only defined at the global image level. 
Therefore, a multi-scale approach as the one proposed here can not be applied to repeatedly refine the tasks at a multitude of scales. 

If there are multiple dense prediction tasks and multiple image level tasks. I would propose that the MTI-Net decoding approach can be applied to solve the dense prediction tasks, while the image level tasks can be tackled by applying global average pooling and a linear layer to the encoder features. 

In summary, I think this is an open question, although one can have some intuition about what is possible for specific task dictionaries.

**Comment by aungriah** [comment]
Thank you very much! Sorry for the late reply
